src/app/services/messenger.service.ts

```diff
@@ -27,12 +27,13 @@ export class MessengerService {
   }
 
   placeOrder(product) {
-    console.log(product)
+    let parameters = new HttpParams().set("userId", Number(localStorage.getItem("userId")));
     this.http.put(this.baseOrderURL, product, {
-      headers: new HttpHeaders({'Content-Type': 'application/json'})
+      headers: new HttpHeaders({'Content-Type': 'application/json'}),
+      params : parameters
     }).subscribe(
-      () => {
-        alert("Order Placed!")
+      (result) => {
+          console.log (result);
       },
```

Comment:
Is this method secured (backend)? If it is user id can be retrieved from the authentication object.

Reply:
Messenger Service is not a great name for the service that places orders. It doesn't convey what it does well.

Reply:
placeOrders probably belongs in the OrderService

Reply:
Yeah that should be in the order service, but the result is just a create response object that we aren't doing anything with at the moment so I think I'm just going to log it somewhere else